packages/web/pages/user/my-account/technologies.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { useTranslation } from 'react-i18next';
-import { FiPlus } from 'react-icons/fi';
+import { FiPlus, FiEye } from 'react-icons/fi';
 import Link from 'next/link';
 import { Protected } from '../../../components/Authorization';
 import { UserProfile } from '../../../components/UserProfile';
 import { DataGrid } from '../../../components/DataGrid';
 import { getUserTechnologies } from '../../../services';
 import { Title } from '../../../components/Common';
+import { IconButton } from '../../../components/Button';
 import { getPeriod } from '../../../utils/helper';
+import { STATUS as dealStatusEnum } from '../../../utils/enums/technology.enums';
+
+export const getDealStatusText = (value) =>
```

Comment:
It's `technologyStatus` here too, isn't it?

Maybe it's worth to create a helper so we don't need to repeat methods like that, to get label text. I did it in buy technology stuff

Reply:
Yeah, it makes sense, but not on this PR 👍